src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -29,7 +29,19 @@ const defaultStyles = {
 
 export default class VictoryScatter extends React.Component {
   static role = "scatter";
-  static supportsTransitions = true;
+
+  static defaultTransitions = {
+    onExit: {
+      duration: 600,
```

Comment:
One thing we might consider for the future is for `onExit.duration` and `onEnter.duration` to support function values.  These functions would accept the total animation duration (the one specified at the top-level), and return a value based on that.  This would be helpful if, for example, you were trying to coordinate animations between two unrelated components, where component A will transition and component B will not.